core-blocks/image/index.js

```diff
@@ -217,15 +218,59 @@ export const settings = {
 			/>
 		);
 
-		return (
-			<figure className={ classes }>
+		const figure = (
+			<Fragment>
```

Comment:
Why not use `<React.Fragment>` instead of importing a package specifically for this purpose?

Reply:
Huh, nice catch. I'm guessing our own `Fragment` predates the React one, but we should look into ditching ours.

Reply:
No, this is correctly implemented as-is. We don't and shouldn't use React directly.

Reply:
Ah, I guess that's so we can control the API surface better?